src/php/server_fpm.rs

```diff
@@ -114,7 +127,7 @@ pub(crate) fn start(php_bin: String) -> (PhpServer, Child) {
              .expect("Could not write to php-fpm config file.");
     }
 
-    let cwd = env::current_dir().unwrap();
+
     let pid_filename = format!("{}/.fpm.pid", cwd.to_str().unwrap());
```

Comment:
I think the PID should also be in the HOME/.rymfony/{hash} directory, to avoid polluting projects and to make sure people don't take care of this in their .gitignore file.

WDYT?

Reply:
It's a good idea.

It's possible to put all logs into `HOME/.rymfony/{hash}/log` when the server is started in background.

Reply:
@Pierstoval I have change the path for all pid file and CGI log.